lib/matplotlib/axes/_axes.py

```diff
@@ -1975,6 +1975,8 @@ def bar(self, left, height, width=0.8, bottom=None, **kwargs):
         if not self._hold:
             self.cla()
         color = kwargs.pop('color', None)
+        if color is None:
```

Comment:
`bar` is not touched in that PR. Only `plot()` and `fill()`, however, that does indirectly impact a lot of other plotting functions since they serve as the basis for many others. I don't think `bar()` calls `fill()`, though.

As for normalizing via kwargs, I'd agree, in general. We would need to be careful on any collisions with other kwargs, and if there are any organically-grown alias-handling logic that would need to be cleaned up as well.